### DISCUSSION TITLE
夜间模式有些问题

### DISCUSSION THREAD

**Discussion by penndu** [ArtalkJS/Artalk#185] [Q&A] [answered]
![Honeycam 2022-06-10 22-27-47](https://user-images.githubusercontent.com/46226528/173087518-ec406efe-d2fe-4d23-ba11-8f222577473a.gif)

**Comment by qwqcode** [comment]
参考：[配置文档](https://artalk.js.org/guide/frontend/config.html#darkmode)

也许是主题适配或是Artalk配置问题

**Reply by penndu** [reply]
> 参考：[配置文档](https://artalk.js.org/guide/frontend/config.html#darkmode)
> 
> 也许是主题适配或是Artalk配置问题

开了darkmode=auto

**Comment by kecrily** [comment, marked as answer]
如果你的主题是通过附加 `class` 来实现深色模式手动切换的话，那么你同时需要修改 Artalk 深色模式的值。

```js
artalkInstance.setDarkMode(true)
```

**Reply by Daleveral** [reply]
请问是在哪一处地方修改这个值呢 ?

**Reply by qwqcode** [reply]
> 请问是在哪一处地方修改这个值呢 ?

https://artalk.js.org/zh/develop/fe-api.html#%E5%A4%9C%E9%97%B4%E6%A8%A1%E5%BC%8F-setdarkmode
